pkg/controller/operators/olm/operator.go

```diff
@@ -271,29 +270,19 @@ func (a *Operator) requeueCSV(name, namespace string) {
 	// We can build the key directly, will need to change if queue uses different key scheme
 	key := fmt.Sprintf("%s/%s", namespace, name)
 	logger := log.WithField("key", key)
-	logger.Debugf("requeueing CSV")
+	logger.Debug("requeueing CSV")
 
-	if queue, ok := a.csvQueues[namespace]; ok {
+	if queue, ok := a.csvQueues[metav1.NamespaceAll]; len(a.csvQueues) == 1 && ok {
```

Comment:
Technically we can't pass any other namespaces when using `NamespaceAll`, but that's only enforced from runtime args and not the Go application code, so it's fine.